astroplan/plots/plots.py

```diff
@@ -0,0 +1,294 @@
+        2) Parallactic angle equation? Move this to separate function?
+        3) observe_timezone -- update with info from observer?
+        4) Dealing with NaN or invalid values.
+
+    Returns an ax object with a parallactic angle vs time plot.
+
+    If an ax object is passed in, plots an additional parallactic angle
+    plot on top.
+    Otherwise, creates a new ax object with a parallactic angle plot.
+
+    Parameters
+    ----------
+    target: `~astroplan.FixedTarget`
+        The celestial body of interest.
+
+    observer: `~astroplan.Observer`
+        The person, telescope, observatory, etc. doing the observing.
+
+    time : `~astropy.time.Time`
+        An collection of times to use in generating data to plot.
+        Can be scalar (or len(time)=1).
+
+    ax : `~matplotlib.axes.Axes` or None
+        Optional.
+
+    style_kwargs : WHAT TYPE IS DICTIONARY? or None
+        Optional.
+        A dictionary with plotting style options.
+
+    dark_plot: Boolean.
+        Optional. False (default) or True.
+    """
+
+    # Set up axes & plot styles if needed.
+    if ax is None:
+        ax = plt.gca()
+    if style_kwargs is None:
+        # Need to add more style options for ax's with existing plots?
+        style_kwargs = {'linestyle': '-', 'color': 'r'}
+    #if dark_plot is True:
+        #Add dark plot stuff here.
+
+    # If Time object is scalar, pad time window.
+    if time.isscalar:
+        time = time + np.linspace(-12, 12, 100)*u.hour
+
+    # Calculate parallactic angle.
+    declination = target.dec
+    latitude = observer.location.latitude
+    azimuth = observer.altaz(time, target).az
+    hour_angle = target.ra
+
+    numerator = np.sin(azimuth) * np.cos(latitude)
+    denominator = np.cos(declination) * (np.cos(azimuth)*np.cos(hour_angle) + np.sin(azimuth)*np.sin(hour_angle)*np.sin(latitude))
+    parallactic_angle = np.arcsin(numerator/denominator)
+
+    # Some checks & info for labels.
+    assert len(time) == len(parallactic_angle)
+
+    if hasattr(observer, 'name') is False:
+        observer_name = 'No Observer/Location Name'
+    else:
+        observer_name = observer.name
+
+    if hasattr(target, 'name') is False:
+        target_name = str('%.3f' % (random.random()))
+    else:
+        target_name = target.name
+
+    observe_date = time[0].datetime.strftime('%Y-%m-%d')
+    observe_timezone = 'UTC'
+
+    # Plot data.
+    ax.plot_date(
+        time.plot_date,
+        parallactic_angle,
+        linestyle=style_kwargs['linestyle'],
+        color=style_kwargs['color'],
+        label=target_name,
+        )
+    ax.figure.autofmt_xdate()
+
+    # Set labels, title, legend, etc.
+    ax.set_ylabel("Parallactic Angle")
+    ax.set_xlabel("Hour - "+observe_timezone)
+    ax.set_title("Parallactic Angle vs Time | "+observer_name+" | "+observe_date+" "+observe_timezone)
+    ax.legend(shadow=True)
+
+    return ax
+
+
+def plot_sky(target, observer, time, ax=None, style_kwargs=None):
```

Comment:
I think you should just delete this function for now, as it's non-functional from what you said in the comments.  No point in having something broken in a PR - you can always save it in a separate branch and just bring it back later.